src/main/java/org/isf/admission/rest/AdmissionController.java

```diff
@@ -212,18 +213,29 @@ public ResponseEntity<List<AdmittedPatientDTO>> allAdmittedPatients()
 	@GetMapping(value = "/admissions/admittedPatients", produces = MediaType.APPLICATION_JSON_VALUE)
 	public ResponseEntity<List<AdmittedPatientDTO>> getAdmittedPatients(
 			@RequestParam(name = "searchterms", defaultValue = "", required = false) String searchTerms,
-			@RequestParam(name = "admissionrange", required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) LocalDateTime[] admissionRange,
-			@RequestParam(name = "dischargerange", required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) LocalDateTime[] dischargeRange)
```

Comment:
this change was made to automatically parse the String parameters sent from the ui, and so avoid all the dates converations below... are you sure you really need the code this way?